float64 th1
float64 th2
float64 d3
float64 th1_dot
float64 th2_dot
float64 d3_dot
---
float64 x
float64 y
float64 z
float64 wx
float64 wy
float64 wz
